src/test/routes/notice-pay.test.ts

```diff
@@ -17,7 +17,7 @@ describe(`on POST ${PageUrls.NOTICE_PAY}`, () => {
       .post(PageUrls.NOTICE_END)
```

Comment:
You're describing a notice_pay test bust then testing a post with notice_end. A successful post with notice_pay should redirect to the average weekly hours page